pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.isort]
+profile = "black"
+line_length = 79
+honor_noqa = true
+src_paths = ["aiobotocore", "tests"]
+
+[tool.black]
+line-length = 79
```

Comment:
man they had to go with 79 chars <sigh>. we may want to increase this by six (await ) to ensure the lines match up

Reply:
we can try, but idk if this will not change for other cases, like lines around 80-85 (alignment will not be consistent in these cases)